xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -201,6 +198,13 @@ def get_cmt_dict(self):
     def get_tile_dict(self):
         return self.tile_dict
 
+    def get_vpr_loc_cmt(self):
+        """
+        Returns a dictionary containing the mapping between
+        VPR physical locations to the belonging clock region
```

Comment:
Please clarify that is the (x, y) location, not (x, y, z).

Reply:
Done